packages/react-html/CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
-## [3.0.1] - 2019-07-04
+## [9.0.1] - 2019-07-04
```

Comment:
The square bracketed text generate hyperlinks when the text is referenced to that link. In this case, we'd need something like the following

```
[9.0.1]: https://github.com/Shopify/quilt/compare/@shopify/react-html@9.0.0...@shopify/react-html@9.0.1
```

Which links to changes from `@shopify/react-html@9.0.0` to @shopify/react-html@9.0.1`. 

You can also reference sewing-kit's, https://github.com/Shopify/sewing-kit/blame/master/CHANGELOG.md#L1902-L1907

Reply:
I thought the square bracket is more of a stylistic choice tbh 😂

Reply:
so I tried it out and didn't love it.

because of the monorepo nature of quilt, comparing between tag will also pick up changes thats does not belong to `react-html` at all 👇 

I think I am just going to remove all the square bracketed

![Screen Shot 2019-07-26 at 11 39 43 AM](https://user-images.githubusercontent.com/1610169/61963796-72f51700-af9a-11e9-9469-62a359740e5c.png)